packages/extension-chains/src/index.ts

```diff
@@ -6,6 +6,7 @@ import type { Chain } from './types';
 
 import { Metadata } from '@polkadot/metadata';
 import { TypeRegistry } from '@polkadot/types';
+import { base64Decode } from '@polkadot/util-crypto';
```

Comment:
So am going to bump util-crypto with single-instance detection on util-crypto as well as metadata (and wasm-crypto, but that is stubbed). This may be an issue after it with useless logs. (Not your issue, and not 100% sure, so just a reminder to myself to check - the issue is that a window object is shared, so it really is wait and see atm...)